.github/workflows/wheels.yml

```diff
@@ -0,0 +1,141 @@
+name: wheels
+
+on:
+  pull_request:
+  push:
+    branches: master
+  release:
+    types: [released, prereleased]
+  workflow_dispatch:  # allows running workflow manually from the Actions tab
+
+jobs:
+
+  build-sdist:
+    runs-on: ubuntu-latest
+
+    env:
+      PIP_DISABLE_PIP_VERSION_CHECK: 1
+
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+
+    - name: Set up Python
+      uses: actions/setup-python@v5
+      with:
+        python-version: '3.x'
+
+    - run: sudo apt-get install -y libmagic1
+
+    - name: Build source distribution
+      run: |
+        pip install --upgrade setuptools wheel pip
+        python setup.py sdist
```

Comment:
This is deprecated.  Please use `build` instead.

Reply:
https://github.com/ahupp/python-magic/pull/294/commits/d3e886c1395422a24ebde5069297359dc8f44e3a